packages/manager/modules/netapp/src/dashboard/template.html

```diff
@@ -59,7 +59,7 @@
                     </oui-tile-definition>
                     <oui-tile-definition
                         term="{{:: 'netapp_dashboard_information_quota' | translate }}"
-                        description="{{ 'netapp_dashboard_information_quota_formatted' | translate:{quota: $ctrl.storage.quota } }}"
+                        description="{{ $ctrl.storage.quota | cucBytes:0:false:'GB' }}"
```

Comment:
manager-filters has been added but we still use `cucBytes` ?

Reply:
Hello @marie-j, please read my comment here:
https://github.com/ovh/manager/pull/6251#discussion_r782844305